src/pyspiffe/svid/x509_svid.py

```diff
@@ -334,36 +328,13 @@ def _write_certs_to_file(
         with open(certs_chain_path, 'wb') as chain_file:
             os.chmod(chain_file.name, _CERTS_FILE_MODE)
             for cert in x509_svid.cert_chain():
-                _write_cert_to_file(cert, chain_file, encoding)
+                write_certificate_to_file(cert, chain_file, encoding)
     except Exception as err:
         raise StoreCertificateError(
-            'Error opening certs chain file: {}'.format(str(err))
+            'Error opening certificates chain file: {}'.format(str(err))
```

Comment:
Error message looks incorrect to me based on the name of the exception. Is it correct?